### PR TITLE
Implement :vsc to run VSCode commands

### PR DESCRIPTION
**Pull request by meluskyc**
**What this PR does / why we need it**:
Add a `:vsc` command to run VSCode commands, including extension commands. 

Examples: 

```
:vsc editor.action.organizeImports
:vsc gitlens.toggleFileBlame
```

As a bonus, we can now map extension commands in vimrc. 

```
noremap <leader>glf :vsc gitlens.showFileHistoryView<CR>
```

**Which issue(s) this PR fixes**
#4463, #4733, probably others. While PR #4750 is another attempt at solving these issues, adding a new command is more in-line with approaches by other Vim emulators. e.g. VSVim uses `:vsc` and Ideavim uses `:action`. 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by J-Fields** [comment]
Thanks for the PR! We do already handle mapping built-in commands via .vimrc with special case logic, but I agree - this seems like the correct approach.

File: src/cmd_line/subparser.ts [hunk 1]
```diff
@@ -348,6 +349,11 @@ export const commandParsers = {
     parser: fileCmd.parseEditNewFileInNewVerticalWindowCommandArgs,
   },
 
+  vsc: {
```
Comment (J-Fields) [inline src/cmd_line/subparser.ts:352]: I think a full name of `vscode` makes sense

Comment (meluskyc) [reply src/cmd_line/subparser.ts]: @J-Fields Updated. Thanks!

File: src/cmd_line/commands/vscode.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }
+
+  public static parse(args: string): VsCodeCommand {
+    const scanner = new Scanner(args);
+    scanner.skipWhiteSpace();
+    return new VsCodeCommand(scanner.isAtEof ? undefined : scanner.nextWord());
+  }
+
+  private async vsc() {
+    await vscode.commands.executeCommand(this.command ?? '');
```
Comment (J-Fields) [inline src/cmd_line/commands/vscode.ts:21]: What does `executeCommand('')` do? I'd think `:vsc` with no argument should just throw `E471`

**Comment by J-Fields** [comment]
Thanks again @meluskyc!

**Pull request merged**